seed/seed.json

```diff
@@ -516,6 +516,33 @@
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"],
                 "country": ["US", "AU", "CA", "FR", "DE", "IE", "JP", "NZ", "GB"]
             }
+        },
+        {
+            "name": "SyncResetProgressTokenStudy",
+            "experiments": [
+                  {
+                      "name": "Enabled",
+                      "probability_weight": 0,
+                      "feature_association": {
+                          "enable_feature": ["ResetProgressMarkerOnCommitFailures"]
+                      }
+                  },
+                  {
+                      "name": "Disabled",
+                      "probability_weight": 100,
+                      "feature_association": {
+                          "disable_feature": ["ResetProgressMarkerOnCommitFailures"]
+                      }
+                  },
+                  {
+                      "name": "Default",
+                      "probability_weight": 0
+                  }
+            ],
+            "filter": {
+                "channel": ["NIGHTLY", "BETA", "RELEASE"],
+                "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
```

Comment:
Note: since this is enabled by default https://github.com/brave/brave-core/blob/06042dcf838745820a5448c4b66bdd017d14c54e/components/sync/engine/brave_model_type_worker.cc#L21 this PR won't disable on iOS (as we haven't rolled out variations to iOS)

Reply:
thanks, I am not expecting lots of affected users on iOS